pkg/metrics/metrics.jsx

```diff
@@ -1824,14 +1842,11 @@ class MetricsHistory extends React.Component {
 
         // on a single machine, cockpit-pcp depends on pcp; but this may not be the case in the beiboot scenario,
         // so additionally check if pcp is available on the logged in target machine
-        if ((cockpit.manifests && !cockpit.manifests.pcp) || this.pmlogger_service.exists === false)
+        if (!this.state.isPythonPCPInstalled || this.pmlogger_service.exists === false)
             return <EmptyStatePanel
                         icon={ExclamationCircleIcon}
-                        title={_("Package cockpit-pcp is missing for metrics history")}
-                        action={this.state.isBeibootBridge === true
-                            // See https://github.com/cockpit-project/cockpit/issues/19143
-                            ? <Text>{ _("Installation not supported without installed cockpit package") }</Text>
-                            : this.state.packagekitExists && <Button onClick={this.handleInstall}>{_("Install cockpit-pcp")}</Button>}
+                        title={_("Python PCP module is missing for metrics history")}
+                        action={this.state.packagekitExists && <Button onClick={this.handleInstall}>{_("Install Python PCP module")}</Button>}
```

Comment:
Let's please not pick on "Python" - just say "PCP is missing". This installs both, and the main thing is really the `pcp` package/service.